spec/features/participants_spec.rb

```diff
@@ -47,48 +47,40 @@
   scenario "logged in as an Organizer I want to be able to sort the participants table by their eating habits" do
     user = FactoryGirl.create(:user)
     profile = FactoryGirl.create(:profile, user: user, last_name: "Peter")
-    application_letter = FactoryGirl.create(:application_letter_accepted, 
+    application_letter = FactoryGirl.create(:application_letter_accepted,
       user: user, event: @event, vegan: true)
     user = FactoryGirl.create(:user)
     profile = FactoryGirl.create(:profile, user: user, last_name: "Paul")
-    application_letter = FactoryGirl.create(:application_letter_accepted, 
+    application_letter = FactoryGirl.create(:application_letter_accepted,
       user: user, event: @event, vegan: true, allergic: true)
     user = FactoryGirl.create(:user)
     profile = FactoryGirl.create(:profile, user: user, last_name: "Mary")
-    application_letter = FactoryGirl.create(:application_letter_accepted, 
+    application_letter = FactoryGirl.create(:application_letter_accepted,
       user: user, event: @event, vegetarian: true)
     user = FactoryGirl.create(:user)
     profile = FactoryGirl.create(:profile, user: user, last_name: "Otti")
-    application_letter = FactoryGirl.create(:application_letter_accepted, 
+    application_letter = FactoryGirl.create(:application_letter_accepted,
       user: user, event: @event, vegetarian: true, allergic: true)
     user = FactoryGirl.create(:user)
     profile = FactoryGirl.create(:profile, user: user, last_name: "Benno")
-    application_letter = FactoryGirl.create(:application_letter_accepted, 
+    application_letter = FactoryGirl.create(:application_letter_accepted,
       user: user, event: @event)
```

Comment:
Yep that block creates a bunch of applications with different names (to distinguish them, as this test is about the sorting functionality of the participants table), which requires different users with profiles. Butmost importantly the applications also have different eating habits (vegetarian, allergic etc; as this test is about sorting by those). So you'd need more than a list different names, but also different eating habits. I don't think you can use a factory that, since you need 3 objects of different classes, but I could write a helper method that takes the name, event, and eating habits as parameter and creates the respective user, profile and application letter for it, if you really think that's necessary. I don't think it would be needed anywhere else.